meson.build

```diff
@@ -75,20 +75,17 @@ endif
 
 #####################################################################
 
-fs = import('fs')
-if get_option('split-usr') == 'auto'
-        split_usr = not fs.is_symlink('/bin')
-else
-        split_usr = get_option('split-usr') == 'true'
+if get_option('split-usr') != 'auto'
+        warning('split-usr is deprecated and will be removed in a future release')
 endif
-if split_usr
-        warning('\n!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!\n\n\n'
-                + '                    split-usr mode is going to be removed\n' +
-                '\n\n!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!')
+if get_option('rootlibdir') != ''
+        warning('rootlibdir is deprecated and will be removed in a future release')
+endif
+if get_option('rootprefix') != ''
+        warning('rootprefix is deprecated and will be removed in a future release')
 endif
```

Comment:
Please use `deprecated: true` in `meson_options.txt` for these (not that we have meson 0.60.0.

Reply:
Or just remove them, considering that they aren't used anymore and aren't mapped to new options. meson should make it an error if any unknown options are passed.

Reply:
I want to use the same CI branches for stable releases, so I'd rather keep the old options around for a couple more releases, less churn for me